pyiron_base/database/filetable.py

```diff
@@ -604,7 +604,9 @@ def _get_job_table(
                     self._job_table.project.str.contains(project_path)
                 ]
             else:
-                return self._job_table[self._job_table.project == project_path]
+                return self._job_table[
+                    self._job_table.project.str.endswith(project_path)
+                ]
```

Comment:
This is the same as enforcing `recursive=True` so it reduces the functionality of pyiron and in particular for the file table this can be dangerous.

Reply:
This is incorrect. If a job has the project "/foo/bar/baz/" and the `project_path` is "/foo/bar/", then `.contains` (for `recursive=True`) will find the project and `.endswith` (for `recursive=False`) will not.